code/modules/mob/living/carbon/brain/brain_emote.dm

```diff
@@ -0,0 +1,65 @@
+/**
+ * Emotes usable by brains, but only while they're in MMIs.
+ */
+/datum/emote/living/carbon/brain
+	mob_type_allowed_typecache = list(/mob/living/carbon/brain)
+	var/self_message
+
+/datum/emote/living/carbon/brain/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	if(!.)
+		return TRUE
```

Comment:
I'm kind of thinking about going through these and replacing them with a #define or something to make the return behavior more clear. I did a lot of work on /tg/ with tool_act procs and it makes it a lot clearer when you have something like `return EMOTE_UNUSABLE`.

not 100% sure though, it might be more trouble than it's worth.